src/js/buildFilterQuery.js

```diff
@@ -28,6 +28,20 @@ export function buildFilterQuery(selectStmt, params) {
 
         const whereObjClause = connector._buildWhere(modelName, safeWhere);
 
+        //add the modelName to each requested field to avoid ambiguity in more complex queries
+        console.log('whereObjClause.sql -------> ', whereObjClause.sql);
+
+        const tableName =
+          modelName.toLowerCase() === 'planterregistration'
+            ? 'planter_registrations'
+            : modelName.toLowerCase();
+        const newQueryFields = whereObjClause.sql
+          .replace(/"/, `"${tableName}.`)
+          .replace(/AND "/gi, `AND "${tableName}.`)
+          .replace(/"/g, '');
+
+        whereObjClause.sql = newQueryFields;
+
```

Comment:
@nmcharlton 
I added this regex to add the modelName back onto each field because our more complex cross-table queries were resulting in "ambiguous" field errors that I couldn't find any other way of resolving.  Is there a better way to resolve those errors?